### ISSUE TITLE
mace_run_train loss output needs more digits

### ISSUE THREAD

**Issue by bernstei** [ACEsuit/mace#646]
loss output in the log that looks like
```
2024-10-18 16:24:36.769 INFO: Epoch 1: head: default, loss=  0.0001, RMSE_E_per_atom=    33.7 meV, RMSE_F=   111.5 meV / A, RMSE_stress=     2.2 meV / A^3
2024-10-18 16:25:29.734 INFO: Epoch 2: head: pt_head, loss=  0.0004, RMSE_E_per_atom=   353.0 meV, RMSE_F=   339.7 meV / A, RMSE_stress=    30.0 meV / A^3
```
does not have enough digits to be useful.  We should either increase the number of digits absolutely, or perhaps (if we're worried about dynamic range) set the number based on the value at the call that generates the `"Initial"` log line

**Comment by birdyLinch** [comment]
To follow up on that, is this loss meant to track the model's convergence? If that's the case, then we might want to consider logging the training loss and always showing RMSE/MAE_E and RMSE/MAE_F, etc., for the validation set since the loss might vary across different runs ;)

**Comment by tisabe** [comment]
I running into this problem pretty quickly.
At the start:
```
2024-09-16 14:17:20.039 INFO: Epoch None: loss=0.0001, MAE_E_per_atom=5.3 meV, MAE_F=20.0 meV / A
2024-09-16 14:25:29.664 INFO: Epoch 0: loss=0.0000, MAE_E_per_atom=0.3 meV, MAE_F=4.4 meV / A
2024-09-16 14:33:25.428 INFO: Epoch 1: loss=0.0000, MAE_E_per_atom=0.2 meV, MAE_F=2.9 meV / A
2024-09-16 14:41:21.101 INFO: Epoch 2: loss=0.0000, MAE_E_per_atom=0.1 meV, MAE_F=2.5 meV / A
```
At the end:
```
2024-09-18 03:21:56.113 INFO: Epoch 196: loss=0.0000, MAE_E_per_atom=0.0 meV, MAE_F=0.6 meV / A
2024-09-18 03:30:05.714 INFO: Epoch 197: loss=0.0000, MAE_E_per_atom=0.0 meV, MAE_F=0.6 meV / A
2024-09-18 03:38:14.386 INFO: Epoch 198: loss=0.0000, MAE_E_per_atom=0.0 meV, MAE_F=0.6 meV / A
2024-09-18 03:46:23.302 INFO: Epoch 199: loss=0.0000, MAE_E_per_atom=0.0 meV, MAE_F=0.6 meV / A
```
After some epochs I cannot really tell how training is progressing.

**Comment by gabor1** [comment]
Yeah we know. We'll increase it to 8 digits oil imminently
